kuma/wiki/views/create.py

```diff
@@ -30,6 +32,23 @@ def create(request):
     # Try to head off disallowed Template:* creation, right off the bat
     if not Document.objects.allows_add_by(request.user, initial_slug):
         raise PermissionDenied
+    # TODO: Integrate this into a new exception-handling middleware
+    if not request.user.has_perm('wiki.add_document'):
+        email_url = (
+            'mailto:%s?'
+            'subject=Page Creation permission&'
+            'body=My username is %s.'
+            ' I want the page creation permission on %s because...'
+        ) % (config.EMAIL_LIST_MDN_ADMINS,
+             request.user.username,
+             request.get_host())
+        context = {
```

Comment:
@safwanrahman asked in IRC if this would be better in the template.

The other existing code that uses an extended ``mailto:`` URL is https://github.com/mozilla/kuma/blob/master/kuma/users/jinja2/users/user_banned.html, setting the subject line.  This example isn't very good, because it sends the entire HTML section, including the URL, to the translators, when instead it should use a placeholder for the URL.  I don't consider this a good example for how to construct a ``mailto:`` link.

A better option would be to assign the URL to a variable, and to use a placeholder in the translation:

```jinja2
{% set request_access_subject="Page Creation permission" %}
{% set request_access_body="My username is %s. I want the page creation permission on %s because..." | format(request.user.username, request.get_host()) %}
{% set request_access_url="mailto:%s?subject=%s&body=%s" | format(request_email_address, request_access_subject, request_access_body) %}
<p class="notice">
  {% trans %}
  Creating a new page is a restricted action.
  To request a new page, please <a href="{{ request_page_url }}">file a bug</a>.
  If you need permission to create new pages, please <a href="{{ request_access_url }}">contact us</a>.
  {% endtrans %}
</p>
```

This looks a lot like the previous version, with some additional variable setting in the template. I prefer the version that constructs the ``mailto:`` string in the view.

This reminds me that I need to regenerate translation strings for this PR, and others, like the new draft functionality, so that the strings are available for translation in Pontoon.  I think this should be done post-merge, as this branch is behind master.